client/dashboard/profile-wizard/style.scss

```diff
@@ -208,12 +204,15 @@
 			justify-content: flex-end;
 
 			.muriel-checkbox label.components-checkbox-control__label {
+				display: inline-block;
 				margin-left: $gap-small;
 			}
 		}
 	}
 
 	.woocommerce-profile-wizard__checkbox-group {
+		margin: -16px -16px 0 -16px;
```

Comment:
We could do `margin: -$gap -$gap 0 -$gap` here.

Reply:
Good call, Justin.  Updated in https://github.com/woocommerce/woocommerce-admin/pull/3156/commits/38d36a241d64ad495ead0147b6e6c72df686cbd0